packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -25,11 +25,11 @@
 
   .header-content {
     .heading {
-      @apply text-n1h;
+      font-size: var(--calcite-font-size--1);
```

Comment:
What's the purpose in swapping these out? Because we're not using tokens for font-size right now we can leave these with tailwind

Reply:
`@apply text-n1h;` Was applying the incorrect line height. 

I can switch those back and overwrite the line height using `line-height: var(--calcite-font-line-height-relative-snug) !important;` in line 131. Would be better?